src/main/java/com/google/sps/servlets/RecommenderServlet.java

```diff
@@ -36,8 +36,7 @@
 public class RecommenderServlet extends HttpServlet {
 
   private final int RECOMMENDATIONS_N = 3;
-  private final int FACTORS_N = 3;
-  private final String[] factors = {"price", "crowd", "distance"};
+  private final String[] FACTORS = {"price", "crowd", "distance"};
```

Comment:
What does distance mean, here? We haven't used distance as a factor.

Reply:
it was supposed to measure the distance between the recommended places and the specific campus, but this is obviously not in scope for  now be removed for now